std-bits/snowflake/src/main/java/org/enso/snowflake/SnowflakeIntegerColumnMaterializer.java

```diff
@@ -0,0 +1,158 @@
+package org.enso.snowflake;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.BitSet;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.numeric.BigIntegerStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.type.BigIntegerType;
+import org.enso.table.data.column.storage.type.IntegerType;
+import org.enso.table.data.column.storage.type.StorageType;
+import org.enso.table.error.ValueTypeMismatchException;
+import org.graalvm.polyglot.Context;
+
+public class SnowflakeIntegerColumnMaterializer extends Builder {
```

Comment:
This logic is similar to `InferredBuilder` but it actually differs. There we don't accept even small `BigInteger` values to be coerced to `long`.

It is also just much simpler than `InferredBuilder` which must handle many more types, so it is better to keep it separate so that its own logic is kept as simple as possible.